proposals/lip-0067.md

```diff
@@ -210,28 +205,25 @@ Secret recovery phrase: `target cancel solution recipe vague faint bomb convince
 ```json
 {
   "crypto": {
-    "version": "1",
+    "cipher": "aes-128-ctr",
+    "cipherparams": {"iv": "da7a74acbf34d20ffd3658f9"},
     "ciphertext": "866c6f1cab3ef67514bdc54cf0143b8b824ebe7c045efb97707c158c81d313cd1a6399b7aa3002248984d39ea2604b0263fe7bdbd8cb04286a9cbd2d353fc79908daab9af04b2528bf4f06a82d79483c",
-    "mac": "a476979ca68fe90f3c96f8a5f3f0a9fe33aef8b091d1169861e44a11a680aae9",
-    "cipher": "aes-256-gcm",
-    "cipherparams": {
-      "iv": "da7a74acbf34d20ffd3658f9",
-      "tag": "f4282899ed6cb0193e2981dca0d2ae8e"
-    },
     "kdf": "argon2id",
     "kdfparams": {
       "parallelism": 4,
       "iterations": 1,
       "memory": 2024,
```

Comment:
I suggest checking if this is the right value for `memory`, or it should be `2048` as mentioned [here](https://github.com/LiskHQ/lips/blob/87e344e4f6f9d6362e958ab1c0fb2d070cbfe6f3/proposals/lip-0067.md?plain=1#L172).

Reply:
Agree with Miroslav

Reply:
Mitsu told me there was a recent change of this value in some PR and the value 2097023kilobytes is the closest value to max 2GB that one can set for memory value. So which value should I include? @mjerkov @gkoumout

Reply:
Update the memory value with 2048.